optuna/terminator/__init__.py

```diff
@@ -16,6 +17,7 @@
     "report_cross_validation_scores",
     "StaticErrorEvaluator",
     "BaseImprovementEvaluator",
+    "BestValueStagnationEvaluator",
```

Comment:
Could you add `BastValueStagnationEvaluator` to the reference, please?

https://github.com/optuna/optuna/blob/master/docs/source/reference/terminator.rst

Reply:
Done